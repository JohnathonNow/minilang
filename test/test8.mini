var test := fun(Start, Stop, Step) do
	var Value := Start
	loop while Value <= Stop
		suspend Value
		Value := old + Step
	end
	return nil
end

for X in test[1, 10, 3] do
	print('X = {X}\n')
end
