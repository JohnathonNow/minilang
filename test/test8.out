X = 1
X = 4
X = 7
X = 10
[1, 8, 15, 22, 29, 36, 43, 50, 57, 64, 71, 78, 85, 92, 99]
